gradle/wrapper/gradle-wrapper.properties

```diff
@@ -5,4 +5,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-bin.zip
```

Comment:
Is this necessary? Might change required java version. I have to check but I think we use the same version in all the repos still.

Reply:
Let me rephrase - if we’re doing it here, I’m assuming we will have to do it across all the repos.